builtin/logical/pkiext/nginx_test.go

```diff
@@ -309,15 +471,77 @@ func RunNginxRootTest(t *testing.T, caKeyType string, caKeyBits int, caUsePSS bo
 		"ip_sans":     "127.0.0.1,::1",
 		"sans":        uniqueHostname + ",localhost,localhost4,localhost6,localhost.localdomain",
 	})
-	requireSuccessNonNilResponse(t, resp, err, "failed to create leaf cert")
+	requireSuccessNonNilResponse(t, resp, err, "failed to create server leaf cert")
 	leafCert := resp.Data["certificate"].(string)
 	leafPrivateKey := resp.Data["private_key"].(string) + "\n"
 	fullChain := leafCert + "\n"
 	for _, cert := range resp.Data["ca_chain"].([]string) {
 		fullChain += cert + "\n"
 	}
 
-	cleanup, host, port, networkName, networkAddr, networkPort := buildNginxContainer(t, fullChain, leafPrivateKey)
+	// Issue a client leaf certificate.
+	resp, err = pki.CBWrite(b, s, "issue/testing", map[string]interface{}{
+		"common_name": "testing.client.dadgarcorp.com",
+	})
+	requireSuccessNonNilResponse(t, resp, err, "failed to create client leaf cert")
+	clientCert := resp.Data["certificate"].(string)
+	clientKey := resp.Data["private_key"].(string) + "\n"
+	clientFullChain := clientCert + "\n" + resp.Data["issuing_ca"].(string) + "\n"
```

Comment:
Should this not have rootCert appended to it as well?

Reply:
I think `clientFullChain` was a bad name. For clients like wget/curl, we provision just the certs that need to be sent on the wire. For nginx/..., they like having the root cert as well, even if they don't send it, so the full chain is the full chain. The same holds for Go, where it'll combine the certs appropriately (taking the leaf separately from its CA chain). 

Maybe `clientWireChain` is better?

Reply:
Yeah that might be better.